parity-util-mem/Cargo.toml

```diff
@@ -11,11 +11,12 @@ cfg-if = "0.1.6"
 malloc_size_of_derive = "0.1.0"
 dlmalloc = { version = "0.1", features = ["global"], optional = true }
 wee_alloc = { version = "0.4", optional = true }
-mimallocator = { version = "0.1", optional = true }
+mimallocator = { version = "0.1", features = ["secure", "stats"], optional = true }
+mimalloc-sys = { version = "0.1", features = ["secure", "stats", "override"], optional = true }
 
 elastic-array = { version = "0", optional = true }
 ethereum-types = { version = "0", optional = true }
-parking_lot = { version = "0", optional = true }
+parking_lot = { version = "0.7", optional = true }
```

Comment:
I think @cheme had a reason for keeping the version very open ended here. You sure you need to nail it down like this?

Reply:
This is a temp workaround for cargo not picking the right version (probably a bug) of parking_lot and failing to compile because of that.

Reply:
Yes, I must admit that I had to set it manually in parity-eth cargo.lock, I do not think the change will break in substrate (already a lot of parking lot different version there), but yes the intent was that parking lot implementation can use multiple version (this is not really a future proof mechanism : breaking change on parking lot would mean having to manage multiple version explicitely which may prove being challenging).